### ISSUE TITLE
Newbie Question: how to determine if mean/std value is suitable

### ISSUE THREAD

**Issue by Cli98** [zylo117/Yet-Another-EfficientDet-Pytorch#25]
Hi author @zylo117 ,

Thank you for your excellent work! I have some questions for mean/std normalization parameters. 

1. Is it for image normalization configuration?
2. Are those set of parameters determined by pretrain model (This is the behavior as described in mmdetection )? If yes, then if we use the same pre-train, then we do not need to modify those parameters right?
3. You also mentioned "mean and std in RGB order, actually this part should **remain unchanged as long as your dataset is similar to coco**". So looks like this also has something to do with dataset itself. I'm not offending, but do you think if this seems conflict with the explanation from mmdetection (namely, determined by model or by dataset)?

Thank you for your response.

**Comment by zylo117** [comment]
1. Yes.

2. No. It's determined by the dataset, but the pretrained model is adapted to this mean/std. You can modify it of course. In my experience, changing mean/std of a pretrained model's preprocessing doesn't help for most of the datasets that comes from daily life. But if your dataset is in a total different domain, you should change it.

3. As in answer 2, it determined by the dataset, no doubt. Because the mean is actually the mean of pixel value of the images. So is std. But since the model had been trained with this kind of preprocessing, the model works best with these parameters.

**Comment by Cli98** [comment]
> * Yes.
> * No. It's determined by the dataset, but the pretrained model is adapted to this mean/std. You can modify it of course. In my experience, changing mean/std of a pretrained model's preprocessing doesn't help for most of the datasets that comes from daily life. But if your dataset is in a total different domain, you should change it.
> * As in answer 2, it determined by the dataset, no doubt. Because the mean is actually the mean of pixel value of the images. So is std. But since the model had been trained with this kind of preprocessing, the model works best with these parameters.

Thank you for your kindly response.
